web/src/ShareSnapshotModal.scss

```diff
@@ -0,0 +1,60 @@
+@import "constants";
+
+.modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width:100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.6);
+}
+
+.modal-main {
+  position:fixed;
```

Comment:
if you've got a fixed element inside another fixed element, that usually means you're confused and not using it correctly. tho i can clean this up in a follow-up pr